spec/acceptance/qpid_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper_acceptance'
+
+describe 'certs' do
+  nssdb_dir = '/etc/pki/katello/nssdb'
+  nssdb_password_file = "#{nssdb_dir}/nss_db_password-file"
+
+  context 'with default params' do
+    let(:pp) do
+      <<-EOS
+      user { 'qpidd':
+        ensure => present,
+      }
+
+      include certs::qpid
+      EOS
+    end
+
+    it_behaves_like 'a idempotent resource'
+
+    describe x509_certificate("/etc/pki/katello/certs/#{host_inventory['fqdn']}-qpid-broker.crt") do
+      it { should be_certificate }
+      it { should be_valid }
+      it { should have_purpose 'server' }
+      include_examples 'certificate issuer', "C = US, ST = North Carolina, L = Raleigh, O = Katello, OU = SomeOrgUnit, CN = #{host_inventory['fqdn']}"
+      include_examples 'certificate subject', "C = US, ST = North Carolina, O = pulp, OU = SomeOrgUnit, CN = #{host_inventory['fqdn']}"
+      its(:keylength) { should be >= 2048 }
+    end
+
+    describe x509_private_key("/etc/pki/katello/private/#{host_inventory['fqdn']}-qpid-broker.key") do
+      it { should_not be_encrypted }
+      it { should be_valid }
+      it { should have_matching_certificate("/etc/pki/katello/certs/#{host_inventory['fqdn']}-qpid-broker.crt") }
+    end
+
+    describe file(nssdb_password_file) do
+      it { should be_file }
+      it { should be_mode 640 }
+      it { should be_owned_by 'root' }
+      it { should be_grouped_into 'qpidd' }
+    end
+
+    describe file(nssdb_dir) do
+      it { should be_directory }
+      it { should be_mode 755 }
+      it { should be_owned_by 'root' }
+      it { should be_grouped_into 'qpidd' }
+    end
+
+    describe command("certutil -L -d #{nssdb_dir}") do
+      its(:exit_status) { should eq 0 }
+      its(:stdout) { should match(/^ca                                                           CT,C,c$/i) }
+      its(:stdout) { should match(/^broker                                                       u,u,u$/i) }
+    end
+
+    describe command("certutil -L -d #{nssdb_dir} -n ca") do
+      its(:exit_status) { should eq 0 }
+      its(:stdout) { should match(/        Subject: "CN=#{host_inventory['fqdn']},OU=SomeOrgUnit,O=Katello,L=Raleig
+            h,ST=North Carolina,C=US"/) }
```

Comment:
I think I'd prefer this if it works.
```suggestion
      its(:stdout) { should match(/\s*Subject: "CN=#{host_inventory['fqdn']},OU=SomeOrgUnit,O=Katello,L=Raleig\n            h,ST=North Carolina,C=US"/) }
```

Reply:
Done, thanks for that suggestion.